requirements/dev.in

```diff
@@ -18,7 +18,7 @@ Scrapy
 https://github.com/andersk/coveragepy/archive/1ccddf9070d26b8837062197423429d2305f6385.zip#egg=coverage==7.2.5+git  # https://github.com/nedbat/coveragepy/issues/1586
 
 # fake for LDAP testing
-fakeldap
+https://github.com/ClundXIII/fakeldap/archive/refs/heads/fix-group-member-names-anded.zip
```

Comment:
This URL is broken.

Reply:
Hi, yes. The branch has been merged and deleted.

That commit will be removed once fakeldap release has been made.